Jenkinsfile

```diff
@@ -348,6 +361,30 @@ def deploy_image_for_testing(arch) {
    templates = null
 }
 
+/* deploy_image_on_production(arch)
+
+   deploys qcow2 images on the various clusters for the given arch.
+   cluster credentials come from packer_pipeline_credentials.json
+   in alfred's home
+
+   TODO: failure while deploying any image is a soft failure
```

Comment:
What's defined as a "soft" failure?

Reply:
I copied this from Samar's function. Soft failure probably means the other images are tested even if this image fails. Currently if an image fails all the tests stop.